app/services/index_info.rb

```diff
@@ -0,0 +1,72 @@
+class IndexInfo
+  DEFAULT_INFO = {
+    state:"not found",
+    num_docs: "not found"
+  }
+  BOOK_INDEX_MATCH = /[\w-]+@\w+.+_+.+/
+
+  def call
+    @book_indexes = {}
+    results
+  end
+
+  private
+
+  def results
+    {
+      es_version: es_version,
+      book_indexes: all_the_books
```

Comment:
Somewhere can we sort the book indexes by creation time (or something).  I'd prefer to know that the results come back in a consistent order on every call to `/info` and I'm unsure if we otherwise have that guarantee.